packages/sdk-browser/src/sdk-client.ts

```diff
@@ -159,28 +159,21 @@ export class TurnkeyBrowserSDK {
 
   logoutUser = async (): Promise<boolean> => {
     await removeStorageValue(StorageKeys.CurrentUser);
-    await removeStorageValue(StorageKeys.AuthBundle);
     await removeStorageValue(StorageKeys.SigningSession);
 
     return true;
   };
 
-  getAuthBundle = async (): Promise<string | undefined> => {
-    return await getStorageValue(StorageKeys.AuthBundle);
-  };
-
   getSigningSession = async (): Promise<SigningSession | undefined> => {
     const signingSession: SigningSession | undefined = await getStorageValue(StorageKeys.SigningSession);
     if (signingSession) {
-      if (signingSession.expirationTimestamp > Date.now()) {
+      if (signingSession.sessionExpiry > Date.now()) {
```

Comment:
`signingSession?.sessionExpiry`?

Reply:
And otherwise we can just return `await removeStorageValue(StorageKeys.SigningSession);`

So in all, something like

```
getSigningSession = async (): Promise<SigningSession | undefined> => {
    const signingSession: SigningSession | undefined = await getStorageValue(StorageKeys.SigningSession);
    if (signingSession?.sessionExpiry > Date.now()) {
      return signingSession;
    }
 
    await removeStorageValue(StorageKeys.SigningSession);
    return;
  }
```

Reply:
I think this is ok the way it is because you aren't calling `removeStorageValue()` unless there's an active session that you need to expoire